react/PartnerSites/Locales/__snapshots__/Locales.test.js.snap

```diff
@@ -3,7 +3,6 @@
 exports[`Locales: should render with locale of AU first and active 1`] = `
 <nav
   aria-labelledby="Locales"
-  class="root"
```

Comment:
It's because this is a non-existent class—a good example of why this is a big improvement.

I opted to leave this in for now, rather than changing production code, since this is supposed to be scoped to tests only.